src/IntercomDefinitions.hs

```diff
@@ -65,19 +67,19 @@ tagName = "foo"
 run :: TVar TopicResult -> IO ()
 run t = void $ do
   options <- opts
-  go $ blank & requestEndpoint .~ "https://api.intercom.io/contacts"
-             & requestOpts .~ options
-             & requestParameters .~ object []
-             & requestTopic .~ "contact.created"
+  contactResp <- go $ blank & requestEndpoint .~ "https://api.intercom.io/contacts"
+                    & requestOpts .~ options
+                    & requestParameters .~ object []
+                    & requestTopic .~ "contact.created"
 
```

Comment:
the above means `contactResp` will hold the response object from the creation request